config.go

```diff
@@ -343,6 +343,19 @@ func (b *Builder) SetWorkDir(there string) {
 	b.Docker.Config.WorkingDir = there
 }
 
+// Shell returns the default shell for running commands in the
+// container, or in a container built using an image built from this container.
+func (b *Builder) Shell() []string {
+	return b.Docker.Config.Shell
+}
+
+// SetShell sets the default shell for running
+// commands in the container, or in a container built using an image built from
+// this container.
```

Comment:
Add a note here like the one we have for `SetHostname()`, so that callers will be less surprised when this doesn't affect the contents of OCI images.

Reply:
Fixed, also added to man page.